tests/storage_test.go

```diff
@@ -321,9 +321,32 @@ var _ = Describe("Storage", func() {
 							virtClient,
 							vmiPod,
 							vmiPod.Spec.Containers[0].Name,
-							[]string{"find", hostDiskDir, "-name", diskName, "-size", "1G"},
+							[]string{"find", hostdisk.GetMountedHostDiskDir("host-disk"), "-name", diskName, "-size", "1G"},
 						)
-						Expect(strings.Contains(output, diskPath)).To(BeTrue())
+						Expect(output).To(ContainSubstring(hostdisk.GetMountedHostDiskPath("host-disk", diskPath)))
+
+						err = virtClient.VirtualMachineInstance(vmi.Namespace).Delete(vmi.Name, &metav1.DeleteOptions{})
```

Comment:
This looks like cleanup, shouldn't this be done in `AfterEach`?

Reply:
let me check.

Reply:
Yep you are right. Moved the whole cleanup sequence into an `AfterEach`.